polaris-react/src/components/MediaCard/MediaCard.tsx

```diff
@@ -91,6 +104,7 @@ export function MediaCard({
             onActionAnyItem={togglePopoverActive}
           />
         </Popover>
+        {dismissButtonMarkup}
```

Comment:
I'd love it if we could keep this outside of the popoverActionsMarkup. That would clear the check on line 160 but might take some css polish to get things looking right. I can help with that part later today / next week if you want

Reply:
I would love a hand on it. Not expecting it to be super difficult, but since it's absolutely positioned i didnt want to pile onto that pattern. 

I originally had a wrapper `styles.TopActions` that `display:flex`ed the dismiss button and the popover activator, maybe that would work better?